exchanges/futures/contract.go

```diff
@@ -14,6 +14,7 @@ import (
 // var error definitions
 var (
 	ErrInvalidContractSettlementType = errors.New("invalid contract settlement type")
+	ErrContractNotSupported          = errors.New("unsupported contract")
```

Comment:
This can be removed as below uses `futures.ErrContractNotSupported`

Reply:
`futures.ErrContractNotSupported` is this.